### DISCUSSION TITLE
Blue running block

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#1155] [Help] [answered]
I have not clear how the addon that highlights in blue the running blocks work. Where and when is the blue highlight supposed to show up? Thanks in advance for any detail

**Comment by cobaltt7** [comment, marked as answer]
The default yellow outline outlines the currently running stack. The blue outline js the currently running block. It only shows up if the block takes a while to execute however, because it would cause too much lag to move the blue outline every few millisecconds.


Paul Reid

________________________________

**Reply by sfederici** [reply]
Thanks for your answer Paul. Even if it would cause too much lag, it could be very useful (not as the default behavior, but as an option) to be able to move the blue outline every few milliseconds. Indeed, when you are learning Scratch, it is very useful to understand the behavior of a script. This is what I do during my lessons with my students by using the very old Scratch 1.4.
